deepchem/models/torch_models/hf_models.py

```diff
@@ -601,3 +603,363 @@ def fill_mask(self,
             results.append(text_results)
 
         return results[0] if len(results) == 1 else results
+
```

Comment:
We need unit tests

Reply:
Added

Reply:
Add a detailed docstring for this method and in the HF model main docstring explaining adaptive bucketing with citation.

Also add a corresponding section in the RST docs.